wagtail/admin/templates/wagtailadmin/pages/add_subpage.html

```diff
@@ -13,14 +13,19 @@
 
         {% if page_types %}
             <ul class="listing">
-                {% for verbose_name, app_label, model_name in page_types %}
+                {% for verbose_name, app_label, model_name, page_description in page_types %}
                     <li>
                         <div class="row row-flush">
-                            <div class="col6">
-                                <a href="{% url 'wagtailadmin_pages:add' app_label model_name parent_page.id %}{% if next %}?next={{ next }}{% endif %}" class="icon icon-plus-inverse icon-larger">{{ verbose_name }}</a>
+                            <div class="col8">
+                                <a href="{% url 'wagtailadmin_pages:add' app_label model_name parent_page.id %}{% if next %}?next={{ next }}{% endif %}" class="icon icon-plus-inverse icon-larger">
+                                    {{ verbose_name }}
+                                    {% if page_description %}
+                                        <div><small>{{ page_description }}</small></div>
```

Comment:
Sorry I didn’t catch this sooner – could you add a:

```suggestion
                                        <div><span class="visuallyhidden">.</span><small>{{ page_description }}</small></div>
```

This will have screen readers have a pause between announcing the name and the description.